ckanext/switzerland/plugins.py

```diff
@@ -200,6 +200,8 @@ def get_helpers(self):
             'ogdch_get_switch_connectome_url': ogdch_backend_helpers.ogdch_get_switch_connectome_url,  # noqa
             'ogdch_get_env': ogdch_backend_helpers.ogdch_get_env,
             'ogdch_multiple_text_form_helper': ogdch_dataset_form_helpers.ogdch_multiple_text_form_helper,  # noqa
+            'get_recaptcha_publickey': ogdch_frontend_helpers.get_recaptcha_publickey,  # noqa
+            'apply_recaptcha': ogdch_frontend_helpers.apply_recaptcha,
```

Comment:
This is good, but like I mentioned in the other PR, I would move both these helpers into ckanext-subscribe.